otter/util/zkpartitioner.py

```diff
@@ -67,9 +67,10 @@ def startService(self):
 
     def stopService(self):
         """Release the buckets."""
-        MultiService.stopService(self)
+        d = MultiService.stopService(self)
```

Comment:
Right, but Partitioner can be declared as `class Partitioner(MultiService, object)`, in which case the new-style class superclass call can be used.

```
>>> class A:
...    def foo(self):
...       print "I am old"
... 
>>> class B(A):
...    def foo(self):
...       super(B, self).foo()
...       print "Should not get here"
... 
>>> B().foo()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 3, in foo
TypeError: must be type, not classobj
>>> class C(A, object):
...    def foo(self):
...       super(C, self).foo()
...       print "I am new"
... 
>>> C().foo()
I am old
I am new
```